envault/cli.py

```diff
@@ -33,13 +35,56 @@ def get_secrets(server, secret, token):
 def cli():
     pass
 
+@cli.command("init")
+@click.option("-profile", help="profile for the config")
+def init(profile):
+    if profile is None:
+        profile = 'default'
+    """ Initialize envault config with vault server, token and secrets path """
+    click.echo("Enter the server, token and path to vault secrets")
+    vault_server = click.prompt('Vault Server', type=str)
+    vault_token = click.prompt('Vault Token', type=str)
+    vault_secret_path = click.prompt('Path to vault secret', type=str)
+
+    click.echo("server %s token %s path %s" %(vault_server, vault_token, vault_secret_path))
+
+    new_profile = {
+        profile: {
+            'vault_server': vault_server,
+            'vault_token': vault_token,
+            'vault_secret_path': vault_secret_path
+        }
+    }
+
+    if os.path.exists('envault.yml'):
+        yaml_file = open('envault.yml', 'r')
+        all_profile_configs = safe_load(yaml_file)
+        if all_profile_configs is not None and profile in all_profile_configs:
+            yaml_file = open('envault.yml', 'w')
+            updated_profile = {
+                **all_profile_configs,
+                **new_profile
+            }
+
+            dump(updated_profile, yaml_file, default_flow_style = False)            
+        else:
+            yaml_file = open('envault.yml', 'a')
+            dump(new_profile, yaml_file, default_flow_style = False)
+    else:
+        yaml_file = open('envault.yml', 'w+')
+        dump(new_profile, yaml_file, default_flow_style = False)
```

Comment:
Also instead of this schema:
```yml
dev:
    vault_secret_path: dev
    vault_server: dev
    vault_token: dev
  default:
    vault_secret_path: default
    vault_server: default
    vault_token: default
```

Let's use this one:
```yml
profiles:
  - name: default
    vault_secret_path: 
    vault_server:
    vault_token:
  - name: dev
    vault_secret_path: 
    vault_server:
    vault_token:
```

this is equivalent to array of objects in a json file.
```json
{
  "profiles": [
       {
         "name": "default",
         "vault_secret_path":  "",
         "vault_server": "",
         "vault_token": ""
    },
    {
       "name": "dev",
       "vault_secret_path": "",
       "vault_server": "",
      "vault_token": ""
    }
  ]
}
```